docs/apm.asciidoc

```diff
@@ -0,0 +1,234 @@
+[id="{p}-apm-server"]
+== APM Server
+
+This section describes how to deploy, configure and access an APM Server with ECK.
+
+* <<{p}-apm-eck-managed-es,Use an Elasticsearch cluster managed by ECK>>
+* <<{p}-apm-advanced-configuration,Advanced configuration>>
+** <<{p}-apm-customize-configuration,Customize the APM Server configuration>>
+** <<{p}-apm-secure-settings,APM Secrets keystore for secure settings>>
+** <<{p}-apm-existing-es,Reference an existing Elasticsearch cluster>>
+** <<{p}-apm-tls,TLS Certificates>>
+* <<{p}-apm-connecting,Connecting to the APM Server>>
+** <<{p}-apm-service,APM Server service>>
+** <<{p}-apm-secret-token,APM Server secret token>>
+
+NOTE: The current Docker image of the APM Server must run as `root` or with the user id 1000. This prevents the APM Server from running in some environments such as OpenShift, or on any Kubernetes cluster that would set a different user in the security context.
+
+[float]
+[id="{p}-apm-eck-managed-es"]
+=== Use an Elasticsearch cluster managed by ECK
+
+When both the APM Server and Elasticsearch are managed by ECK it allows a smooth and secured integration between the two. The output configuration of the APM Server is setup automatically to establish a trust relationship with Elasticsearch.
+
+. If you want to deploy an APM Server and connect it to the cluster `quickstart` created in the link:k8s-quickstart.html[quickstart] then apply the following specification:
```

Comment:
To deploy an APM Server and connect it to the cluster `quickstart` created in the link:k8s-quickstart.html[quickstart], apply the following specification: